src/components/DpSearchField/DpSearchField.vue

```diff
@@ -1,15 +1,27 @@
 <template>
-  <span :class="{ 'inline-block w-full': inputWidth !== ''}">
+  <span
+    class="inline-flex"
+    :class="{ 'w-full': inputWidth !== ''}">
     <dp-resettable-input
       id="searchField"
       data-cy="searchField"
       :class="cssClasses"
-      :input-attributes="{ placeholder: translations.search, type: 'search' }"
+      :input-attributes="{ placeholder: Translator.trans('search'), type: 'search'}"
```

Comment:
why do You wan't to use the translator here?
We are working on removing the whole translator in demosplan-ui. So please keep it the way it was or pass the translated text as prop :pray:

Reply:
...and that's why you shouldn't copy/paste. Changed in 01e6c4d4d2a590291b0dffb2a5165249ae199b67